src/components/Input/Input.js

```diff
@@ -1,13 +1,27 @@
-import './Input.css'
+import "./Input.css";
 
-const Input = ({type, placeholder, value, className, handleChange, name}) => {
+const Input = ({
```

Comment:
You can use this syntax => `const Input = (props) => { return <input {...props} /> }`